### PR TITLE
feat(private-database): add link to configuration documentation

### PR DESCRIPTION
**Pull request by Meuh-42**
Signed-off-by: Fabien Bagard <fabien.bagard@ovhcloud.com>

| Question         | Answer
| ---------------- | ---
| Branch?          | `develop` 
| Bug fix?         | no
| New feature?     | no
| Breaking change? | no
| Tickets          | PUDBLOW-1496 
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [x] Breaking change is mentioned in relevant commits

## Description

As there is many options, add a link to docs related to these options

## Related

- [x] TRANS-42909

### CODE REVIEW

File: packages/manager/apps/web/client/app/private-database/dashboard/translations/Messages_fr_FR.json [hunk 1]
```diff
@@ -60,6 +60,7 @@
   "privateDatabase_configurations_general_postgresql": "Configuration générale de PostgreSQL",
   "privateDatabase_configurations_general_postgresql_soon": "Bientôt disponible pour PostgreSQL",
   "privateDatabase_configuration_alert": "Toute modification entraîne un redémarrage du serveur.",
+  "privateDatabase_configuration_documentation": "Plus d'informations sur ces options : <a href=\"https://docs.ovh.com/fr/hosting/configurer-optimiser-son-serveur-de-base-de-donnees/#modifier-la-configuration-de-mon-serveur-de-bases-de-donnees\">https://docs.ovh.com/</a>",
```
Comment (frenautvh) [inline packages/manager/apps/web/client/app/private-database/dashboard/translations/Messages_fr_FR.json:63]: the link should be a parameter passed using angular-translate-values and should be different depending on user country

**Review by frenautvh — CHANGES_REQUESTED** [review]

File: packages/manager/apps/web/client/app/private-database/dashboard/translations/Messages_fr_FR.json (hunk 1 above)
Comment (ghyenne) [reply packages/manager/apps/web/client/app/private-database/dashboard/translations/Messages_fr_FR.json:63]: Ideally also output the html from the translation file.

File: packages/manager/apps/web/client/app/private-database/configuration/private-database-configuration.html [hunk 2]
```diff
@@ -31,6 +31,12 @@
             ></h2>
 
             <p data-translate="privateDatabase_configuration_alert"></p>
+            <a
+                data-translate="privateDatabase_configuration_documentation"
+                href="{{:: configurationsCtrl.configuration_guide }}"
```
Comment (antleblanc) [inline packages/manager/apps/web/client/app/private-database/configuration/private-database-configuration.html:36]: **suggestion:** We can add `target="_blank"` and `rel="noopener"`

File: packages/manager/apps/web/client/app/private-database/configuration/private-database-configuration.html [hunk 3]
```diff
@@ -31,6 +31,12 @@
...
+            >
+            </a>
+            <p></p>
```
Comment (antleblanc) [inline packages/manager/apps/web/client/app/private-database/configuration/private-database-configuration.html:39]: **question:** Why using an empty `<p>` element?

File: packages/manager/apps/web/client/app/private-database/configuration/private-database-configuration.html (hunk 2 above)
Comment (Meuh-42) [reply packages/manager/apps/web/client/app/private-database/configuration/private-database-configuration.html]: Applied

File: packages/manager/apps/web/client/app/private-database/configuration/private-database-configuration.html (hunk 3 above)
Comment (Meuh-42) [reply packages/manager/apps/web/client/app/private-database/configuration/private-database-configuration.html]: Changed by a <div>

**Review by antleblanc — DISMISSED** [review]

**Review by wdhif — DISMISSED** [review]

**Review by JeremyDec — DISMISSED** [review]

**Comment by antleblanc** [comment]
Pulling translation.

**Review by antleblanc — APPROVED** [review]

**Pull request merged**